pkg/commands/workload_get.go

```diff
@@ -157,9 +161,20 @@ func (opts *WorkloadGetOptions) Exec(ctx context.Context, c *cli.Config) error {
 		c.Infof("Supply Chain resources not found.\n")
 	}
 
+	// Print workload status
+	c.Printf("\n")
+	c.Boldf("Issues\n")
+	if workloadStatusReadyCond != nil && workloadStatusReadyCond.Message != "" {
+		if err := printer.WorkloadIssuesPrinter(c.Stdout, workloadStatusReadyCond, workload); err != nil {
```

Comment:
Yeah, so that was something I wasn't so sure to do, but it was either do this, or query the workload twice (in `workload get` file and also in the `WorkloadIssuesPrinter` func) to obtain the condition. I went with this after double checking with Diego but it's just an option. If maybe you have a better idea, something that'd be cleaner, please let me know. @shashwathi

Reply:
```suggestion
		if workloadStatusReadyCond != nil {
		if err := printer.WorkloadIssuesPrinter(c.Stdout, workload); err != nil {
			return err
		}
	} else {
		c.Infof("No issues reported.\n")
	}

```

Reply:
I need to decide whether the info message is going to be shown or if there is an actual issue with the workload.
do you think the validation would be better as 

```go
workloadStatusReadyCond != nil  && workloadStatusReadyCond.Message != "" {
   ...
}
```

Or

```go
workloadStatusReadyCond != nil && (workloadStatusReadyCond.Status == "False" or workloadStatusReadyCond.Status == "Unknown") {
   ...
}
```

Reply:
All these checks can be done in `WorkloadIssuesPrinter` func, you have access to ready condition in there.

Reply:
I would actually keep it in `workload_get`, because the `No issues reported` line is an info message, and I do not have acces to the `cli.Config` inside `WorkloadIssuesPrinter`